imports/api/activities/activity.methods.js

```diff
@@ -24,6 +26,77 @@ Meteor.methods({
     }
   },
 
+  getAllOccurences () {
```

Comment:
what should we do ? I can bind BookingsField file data withTracker like in Calendar if you want. ??

Reply:
I was thinking it would be resource-efficient to do the calculations in the client so that the server doesn't do all this. Also feels like `parseActsWithResources` does kind of the same thing?

Maybe it's best to have a shared state in between all of these components that take in all the activities and parse/use them somehow. Like you say, few different pages fetch them all on page load, and instead it could it be a simple, shared, efficient way to handle this.

For now, let's leave it, until we find an ideal way to do it